infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServerCache.java

```diff
@@ -144,4 +132,42 @@ public void invalidateAll() {
     public int getSize() {
         return dataCache.asMap().size();
     }
+
+    @Builder
+    public static class DataCacheConfigurator {
```

Comment:
Can you change this to `DataCacheConfiguration` instead?

Reply:
done